movement/napari/loader_widgets.py

```diff
@@ -110,6 +143,19 @@ def _create_file_path_widget(self):
         self.file_path_layout.addWidget(self.browse_button)
         self.layout().addRow("file path:", self.file_path_layout)
 
+    def _on_source_software_changed(self, current_text: str):
+        """Enable/disable the fps spinbox based on source software."""
+        is_netcdf = current_text in SUPPORTED_NETCDF_FILES
+        # Disable fps box if netCDF
+        self.fps_spinbox.setEnabled(not is_netcdf)
+
+        if is_netcdf:
+            self.fps_spinbox.setToolTip(
+                "FPS is read directly from the netCDF file."
```

Comment:
Let's spell it as `fps` (lowercase), to match how it's shown in the widget.
Also it's good to be explicit about where exactly the value is taken from (the file attributes).
```suggestion
                "The fps (frames per second) is read directly \n"
                "from the netCDF file attributes."
```